upload_to_s3.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e -o pipefail
+
+reponame=$1
+vpc=$2
+environment=$3
+version=$4
+
+aws s3 cp --recursive ./src/ s3://qmg-website-assets/$reponame/$vpc/$environment/$version
```

Comment:
as there is no environment specific config it should upload like this
```suggestion
aws s3 cp --recursive ./src/ s3://qmg-website-assets/$reponame/$version
```

Reply:
But we still need environment uat / prod? so I take VPC out?

Reply:
done